# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/normalize/hotfix-ftruncate.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- normalize-0.7.7/nid3lib/write.c.orig
+++ normalize-0.7.7/nid3lib/write.c
@@ -59,15 +59,6 @@
 #if HAVE_INTTYPES_H
 # include <inttypes.h>
 #endif
-#if HAVE_FTRUNCATE
-/*
- * ftruncate() shouldn't be declared yet, since we turn on
- * _POSIX_C_SOURCE, but you never know, so we use an old style
- * declaration to lessen the chance of conflict.
- */
-int ftruncate();
-#endif
-
 #include "nid3P.h"

 #define USE_TEMPFILE 1
